pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -291,6 +295,20 @@ def test_task_progress_report(self):
                     0
                 )
 
+    def test_unit_count_on_repo(self):
+        """Verify that the sync added the correct number of units to the repo.
+
+        Looks at the content counts on the repo.
+
+        This also verifies that the counts themselves are getting set on the
+        repo.
+        """
+        counts = self.repos_iter[0].get('content_unit_counts', {})
+        self.assertEqual(counts.get('rpm'), 32)
+        self.assertEqual(counts.get('erratum'), 4)
+        self.assertEqual(counts.get('package_group'), 2)
+        self.assertEqual(counts.get('package_category'), 1)
```

Comment:
Gotcha. In that case, I guess hard-coded numbers should be OK. Let's just reference where they're coming from.